File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/duckdb/TimestampPrecision.scala
package dev.typr.foundations.docs.duckdb
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*


import java.time.LocalDateTime

@SuppressWarnings(Array("unused"))
object TimestampPrecision:
  //start
  val tsSeconds: DuckDbType[LocalDateTime] = DuckDbTypes.timestamp_s
  val tsMillis: DuckDbType[LocalDateTime] = DuckDbTypes.timestamp_ms
  val tsNanos: DuckDbType[LocalDateTime] = DuckDbTypes.timestamp_ns
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/duckdb/TimestampPrecision.java
package dev.typr.foundations.docs.duckdb;

import dev.typr.foundations.DuckDbType;
import dev.typr.foundations.DuckDbTypes;

import java.time.LocalDateTime;

@SuppressWarnings("unused")
public class TimestampPrecision {
    //start
    DuckDbType<LocalDateTime> tsSeconds = DuckDbTypes.timestamp_s;
    DuckDbType<LocalDateTime> tsMillis = DuckDbTypes.timestamp_ms;
    DuckDbType<LocalDateTime> tsNanos = DuckDbTypes.timestamp_ns;
    //stop
}
